File: src/main/java/com/example/dto/account/AccountShortCutDto.java
package com.example.dto.account;

import com.example.dto.image.ImageDto;
import lombok.*;
import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AccountShortCutDto {
    private UUID id;
    private String nickname;
    private String login;
    private ImageDto image;
}

File: src/main/java/com/example/dto/account/AccountShortcutDto.kt
package com.example.dto.account

import com.example.dto.image.ImageDto
import org.springframework.data.redis.core.RedisHash
import java.util.UUID

@RedisHash("accounts")
data class AccountShortcutDto(
    val uuid: UUID,
    val nickname: String?,
    val login: String,
    val image: ImageDto?
)
